test/runAllTests.cmd

```diff
@@ -0,0 +1,8 @@
+%DNX_HOME%\runtimes\dnx-clr-win-x86.1.0.0-beta4\bin\dnx.exe .\Microsoft.ApplicationInsights.AspNet.Tests test
```

Comment:
+1 for explaining DNX_HOME in the script. 
I'd prefer to start out with PowerShell now, before we bump into an advanced scripting task that's hard or impossible to do with Windows shell scripts. Downloading DNX is a nice example. Besides, I heard PoSh is what the cool kids do these days :sunglasses: